vulcan/lib/server/controllers/sessions_controller.rb

```diff
@@ -154,5 +154,17 @@ def send_headers(stream)
   def can_hijack?
     @request.env['rack.hijack?']
   end
+
+  def token
+    # For development, we can inject a production token
+    #   via config.yml, to talk directly to production services.
+    #   This should reduce duplication of data.
+    # Note: You'll need to configire the Magma host to
+    #   in config.yml :development to also point to
+    #   production
+    return @user.token if :production == Vulcan.instance.environment
```

Comment:
I think it is dangerous to have the production behavior running behind a guard clause; better to check for the exceptional environment (:development) and isolate that, so the fallback behavior is production.

Reply:
Yes, I already made that change in the updated version of this.

Reply:
Derp